TweetMate/src/main/java/com/ybsystem/tweetmate/models/entities/twitter/TwitterStatus.java

```diff
@@ -254,4 +265,34 @@ private static String createVia(Status status) {
         }
     }
 
+    private static ArrayList<String> createImageUrls(Status status) {
+        // Init
+        ArrayList<String> imageUrls = new ArrayList<>();
+        Status source = status.isRetweet() ? status.getRetweetedStatus() : status;
+
+        // Check media
+        if (source.getMediaEntities().length > 0) {
+            // Twitter media
+            MediaEntity[] medias = source.getMediaEntities();
+            for (int i = 0; i < medias.length; i++) {
+                imageUrls.add(medias[i].getMediaURLHttps());
+            }
+        } else {
+            // Third party media
+            for (URLEntity url : source.getURLEntities()) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS135)

Reply:
Pending